### ISSUE TITLE
preprocessor: true и пустые правила

### ISSUE THREAD

**Issue by exdeniz** [htmlhero/grunt-init-block#4]
При использование препроцессора и автоматической сборки через watch stylus начинает ругатся на пустые блоки. Было бы отлично, если возможно вставить опцию типа:

```
options:
     preprocessor_rule: 'empy()'
```

**Comment by htmlhero** [comment]
Да, стилус ругается на пустые блоки, но я не очень понял, как должна работать предлагаемая опция. Можете рассказать подробней?

**Comment by exdeniz** [comment]
При использование препроцессора вставлять пустое правило в селектор типа font: 100% или empty()
